src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -377,57 +420,36 @@ private void processExports(Node script, String moduleName) {
         // One top-level assign: transform to
         // moduleName = rhs
         Node ref = moduleExportRefs.get(0);
-        Node newName = IR.name(moduleName);
+        Node newName = IR.name(moduleName).srcref(ref);
         newName.putProp(Node.ORIGINALNAME_PROP, ref.getQualifiedName());
+        Node rhsValue = ref.getNext();
 
-        Node rhsValue = ref.getNext().detachFromParent();
-        Node newExprResult = IR.exprResult(IR.assign(newName, rhsValue)
-          .useSourceInfoIfMissingFromForTree(ref.getParent()));
-
-        // If the rValue is an object literal, check each property to see if
-        // it's an alias, and if it is, copy the annotation over.
-        // This is a common idiom to export a set of constructors.
         if (rhsValue.isObjectLit()) {
-          Scope globalScope = SyntacticScopeCreator.makeUntyped(compiler)
-              .createScope(script, null);
-          for (Node key = rhsValue.getFirstChild();
-               key != null; key = key.getNext()) {
-            if (key.getJSDocInfo() == null &&
-                (key.getFirstChild() == null || key.getFirstChild().isName())) {
-              String aliasedVarName = key.getFirstChild() == null ?
-                  key.getString() : key.getFirstChild().getString();
-              Var aliasedVar = globalScope.getVar(aliasedVarName);
-              JSDocInfo info =
-                  aliasedVar == null ? null : aliasedVar.getJSDocInfo();
-              if (info != null &&
-                  info.getVisibility() != JSDocInfo.Visibility.PRIVATE) {
-                key.setJSDocInfo(info);
-              }
-            }
-          }
+          addConstToObjLitKeys(rhsValue);
         }
 
         Node assign = ref.getParent();
-        Node exprResult = assign.getParent();
-        script.replaceChild(exprResult, newExprResult);
+        assign.replaceChild(ref, newName);
+        JSDocInfoBuilder builder = new JSDocInfoBuilder(true);
+        builder.recordConstancy();
```

Comment:
It's definitely disabled - and enabling looks to be a non-trivial amount of work.